### ISSUE TITLE
Validation Generagtor Issue in CSV Gen

### ISSUE THREAD

**Issue by TaeseongMoon** [fizyr/keras-retinanet#1488]
Hi,

I tried training with my custom data. 
I add --val_annotaions option with CSV Gen. After 16~21 epochs, I got this
`
100/100 [==============================] - 47s 469ms/step - loss: 1.8987 - regression_loss: 1.5175 - classification_loss: 0.3812
2020-11-20 02:09:48.998003: W tensorflow/core/kernels/data/generator_dataset_op.cc:103] Error occurred when finalizing GeneratorDataset iterator: Failed precondition  : Python interpreter state is not initialized. The process may be terminated.
         [[{{node PyFunc}}]]
`
Here is my command, 
```
keras_retinanet/bin/train.py --batch-size 1 --steps 100 --snapshot-path checkpoint/12_46 \
--tensorboard-dir logs/12_45 --epochs 50 --gpu 0 csv train.csv class.csv --val_annotations val.csv
```

Dataset split is train - 4000, val-500.
disk capacity, available cpu were enough
Please give me solution.

**Comment by jackie-angelswing** [comment]
@TaeseongMoon  Did you manage to resolve this issue?

**Comment by sindhura234** [comment]
@TaeseongMoon @jackie-angelswing Did you resolve this issue?

**Comment by jackie-angelswing** [comment]
@sindhura234 I ended up rolling back the version of repo and dependencies, and I don't experience the same anymore. But maybe there is a more permanent solution to the problem? 
keras-resnet==0.1.0
keras==2.3.1
tensorflow==2.2.1

**Comment by omiderfanmanesh** [comment]
same problem ...
